serenity-model/src/main/java/net/thucydides/core/logging/ConsoleLoggingListener.java

```diff
@@ -272,17 +272,7 @@ public void stepStarted(ExecutedStepDescription description) {
         nestedSteps.push(description.getName());
         if (loggingLevelIsAtLeast(LoggingLevel.VERBOSE)) {
             String indent = indentation(nestedSteps.size());// StringUtils.repeat("  ", nestedSteps.size());
-            System.out.println(withTimestamp(colored.green(indent + description.getTitle())));
-//            getLogger().info(colored.green(indent + description.getTitle()));
-        }
-    }
-
-    private String withTimestamp(String message) {
-        if (environmentVariables.getPropertyAsBoolean("serenity.console.timestamp", false)) {
-            String timeStamp = new SimpleDateFormat("HH:mm:ss.SSS").format(new Date());
-            return timeStamp + " " + message;
-        } else {
-            return message;
+            getLogger().info(colored.green(indent + description.getTitle()));
```

Comment:
This will disable the `serenity.console.timestamp` property.

Reply:
Maybe I can add serenity.console.timestamp.format?